no-new-envelopes-alert.tf

```diff
@@ -0,0 +1,23 @@
+module "no-new-envelopes-alert" {
+  source            = "git@github.com:hmcts/cnp-module-metric-alert"
+  location          = "${azurerm_application_insights.appinsights.location}"
+  app_insights_name = "${azurerm_application_insights.appinsights.name}"
+
+  enabled     = "${var.env == "prod"}"
+  alert_name  = "No_new_envelopes_-_Bulk_Scan_Processor"
+  alert_desc  = "Triggers when Bulk Scan Processor did not receive new envelopes"
+
+  app_insights_query = <<EOF
+traces
+| where message startswith "No envelopes received"
+EOF
+
+  frequency_in_minutes        = 60
+  time_window_in_minutes      = 90
```

Comment:
is time window supposed to be 60?

Reply:
i think increased time window is ok. perhaps 15min is enough but i don't mind going with +30. borderline